.cargo/config.toml

```diff
@@ -1,12 +1,14 @@
+# Remove "-Ctarget-feature=+avx2" if your x86_64 target does not support AVX2 vector extensions
 [target.x86_64-unknown-linux-gnu]
 rustflags = [
-    "-Clink-arg=-fuse-ld=lld", "-Clink-arg=-Wl,--no-rosegment", "--cfg", "tokio_unstable"
+    "-Clink-arg=-fuse-ld=lld", "-Clink-arg=-Wl,--no-rosegment", "-Ctarget-feature=+avx2", "--cfg", "tokio_unstable"
```

Comment:
We require this as we want the default to be to build with intrinsics. This is because in the docker case, we may be cross-compiling for another platform, and we want the default to be to include SIMD instrinsics. We do not want the build to be dependent on the build machine.

Reply:
Furthermore, setting `target-cpu=native` may result in an incompatible build for target machine as the compiler may include other extensions or intrinsics available on build machine but not on target. So we want to narrow down specifically to `-Ctarget-feature=+avx2`.

Perhaps we simply need to provide a way to disable SIMD flag for docker. To do so, we will simply have a script that replaces the .cargo/config.toml file with a non-simd version.

Reply:
Done.